src/main/kotlin/no/nav/syfo/cronjob/CronjobModule.kt

```diff
@@ -74,14 +79,25 @@ fun Application.cronjobModule(
     val kafkaDialogmoteStatusEndringProducer = KafkaProducer<String, KDialogmoteStatusEndring>(
         kafkaDialogmoteStatusEndringProducerProperties
     )
+    val kafkaDialogmotesvarProducerProperties = kafkaDialogmotesvarProducerConfig(environment.kafka)
 
+    val kafkaDialogmotesvarProducer = KafkaProducer<String, KDialogmotesvar>(
+        kafkaDialogmotesvarProducerProperties
+    )
```

Comment:
For en som ikke er så god på Kafka: Forstår jeg det rett at en KafkaProducer "produserer" data på en topic? Og så har man sikkert en konsument av noe slag et annet sted?

Reply:
Ja, det stemmer.